app/styles/_variables.scss

```diff
@@ -1,6 +1,6 @@
 $primary_color: rgb(51, 85, 139);
 $secondary_color: rgb(230, 204, 34);
-$black: rgb(0,0,0);
+$black: #686469;
```

Comment:
This color results in a non-conformant color contrast: 

"Element has insufficient color contrast of 2.49 (foreground color: #686469, background color: #a9aaab, font size: 13.5pt (18px), font weight: bold). Expected contrast ratio of 4.5:1"

If we can find a "less black" black that will pass color contrast checks, that would be fine to change.